src/structures/response/IAdminWhois.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * From https://matrix.org/docs/spec/client_server/r0.5.0#get-matrix-client-r0-admin-whois-userid
+ */
+export interface IAdminWhois {
```

Comment:
these interfaces would also be exported from the admin APIs class, like the group profile interface in unstableApis